### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca runcom@redhat.com

### CODE REVIEW

**Comment by runcom** [comment]
ping @cgwalters

File: eap-latest-setup.md [hunk 1]
```diff
@@ -1302,7 +1302,7 @@ few moments (it may take up to a few minutes):
     NAME              READY     REASON    RESTARTS   AGE
     router-1-deploy   1/1       Running   0          21s
 
-In the above router creation command (`oadm router...`) we also specified
+In the router creation command (`oadm router...` in the above openshift docs) we also specified
```
Comment (cgwalters) [inline eap-latest-setup.md]: s/openshift/OpenShift/ ?

Comment (runcom) [inline eap-latest-setup.md]: fixed

**Comment by cgwalters** [comment]
:+1:

**Pull request merged**